volumes/vds/py/vds_functions.py

```diff
@@ -23,19 +23,6 @@ def check_dst(start_date):
         LOGGER.info(f"Normal day.")
     return tz_1 == 'EDT' and tz_2 == 'EST'
 
-def fetch_pandas_df(conn, query, table_name):
-    #generic function to pull data in pandas dataframe format 
-    try:
-        with conn.get_conn() as con:
-            LOGGER.info(f"Fetching {table_name}")
-            data = pd.read_sql(query, con)
```

Comment:
This should still work fine if this query is string. But I agree it's better to delete the whole function as long as it's no longer being used